packages/web-components/src/slider/slider.spec.ts

```diff
@@ -0,0 +1,593 @@
+import { expect, test } from '@playwright/test';
+import type { Locator, Page } from '@playwright/test';
+import { fixtureURL } from '../helpers.tests.js';
+import { Direction } from '@microsoft/fast-web-utilities';
+import type { Slider } from './slider.js';
+
+test.describe('Slider', () => {
+  let page: Page;
+  let element: Locator;
+  let root: Locator;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+
+    element = page.locator('fluent-slider');
+
+    root = page.locator('#root');
+
+    await page.goto(fixtureURL('components-slider--slider'));
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  test('should set and retrieve the `size` property correctly', async () => {
+    await element.evaluate((node: Slider) => {
+      node.size = 'small';
+    });
+
+    await expect(element).toHaveJSProperty('size', 'small');
+
+    await element.evaluate((node: Slider) => {
+      node.size = 'medium';
+    });
+
+    await expect(element).toHaveJSProperty('size', 'medium');
+  });
+
+  // Foundation tests
+  test('should have a role of `slider`', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+            <fluent-slider></fluent-slider>
+        `;
+    });
+    await expect(element).toHaveAttribute('role', 'slider');
+  });
+
+  test('should set a default `min` property of 0 when `min` is not provided', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+            <fluent-slider></fluent-slider>
+        `;
+    });
+
+    await expect(element).toHaveJSProperty('min', 0);
+  });
+
+  test('should set a default `max` property of 10 when `max` is not provided', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+            <fluent-slider></fluent-slider>
+        `;
+    });
+
+    await expect(element).toHaveAttribute('max', '10');
```

Comment:
Are you looking for attribute or JS Property?

Reply:
Good catch, will fix!